<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1026</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_Consult">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(190,209,216);</string>
    </property>
    <property name="text">
     <string>Consultar</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_Consult">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>331</width>
      <height>401</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(222,232,235);</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Consults">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(190,209,216);</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_Accion">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>331</width>
      <height>401</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(222,232,235);</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Accion">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>490</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(190,209,216);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Accion">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(190,209,216);</string>
    </property>
    <property name="text">
     <string>Activar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Generate">
    <property name="geometry">
     <rect>
      <x>1730</x>
      <y>20</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(190,209,216);</string>
    </property>
    <property name="text">
     <string>Generar</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Generate">
    <property name="geometry">
     <rect>
      <x>1820</x>
      <y>20</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(190,209,216);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Consult">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>311</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(222,232,235);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Action">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>311</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(222,232,235);</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_Consult">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(190,209,216);</string>
    </property>
    <property name="maximum">
     <number>100000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_Action1">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>490</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(190,209,216);</string>
    </property>
    <property name="minimum">
     <number>-1</number>
    </property>
    <property name="maximum">
     <number>100000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_Action2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>490</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(190,209,216);</string>
    </property>
    <property name="minimum">
     <number>-1</number>
    </property>
    <property name="maximum">
     <number>9</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1921</width>
      <height>1061</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_Generate">
    <property name="geometry">
     <rect>
      <x>1760</x>
      <y>60</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(190,209,216);</string>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100000</number>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_Generate">
    <property name="geometry">
     <rect>
      <x>1755</x>
      <y>100</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(222,232,235);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Info">
    <property name="geometry">
     <rect>
      <x>1790</x>
      <y>850</y>
      <width>101</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_Info{
background-color:transparent;
border-image:url(:/Images/Images/Logo.png);
background:none;
border:none;
background-repeat:none;
}
#pushButton_Info:checked
{
 border-image:url(:/Images/Images/Logo.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_Winner">
    <property name="geometry">
     <rect>
      <x>805</x>
      <y>20</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(222,232,235);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_GIFS">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>330</y>
      <width>521</width>
      <height>391</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_background</zorder>
   <zorder>pushButton_Consult</zorder>
   <zorder>listWidget_Consult</zorder>
   <zorder>comboBox_Consults</zorder>
   <zorder>listWidget_Accion</zorder>
   <zorder>comboBox_Accion</zorder>
   <zorder>pushButton_Accion</zorder>
   <zorder>pushButton_Generate</zorder>
   <zorder>comboBox_Generate</zorder>
   <zorder>lineEdit_Consult</zorder>
   <zorder>lineEdit_Action</zorder>
   <zorder>spinBox_Consult</zorder>
   <zorder>spinBox_Action1</zorder>
   <zorder>spinBox_Action2</zorder>
   <zorder>spinBox_Generate</zorder>
   <zorder>listWidget_Generate</zorder>
   <zorder>pushButton_Info</zorder>
   <zorder>listWidget_Winner</zorder>
   <zorder>label_GIFS</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCOnfig">
    <property name="title">
     <string>Config</string>
    </property>
    <addaction name="actionTxtFiles"/>
    <addaction name="actionJudgment"/>
   </widget>
   <addaction name="menuCOnfig"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionTxtFiles">
   <property name="text">
    <string>Text Files</string>
   </property>
  </action>
  <action name="actionJudgment">
   <property name="text">
    <string>Judgment</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
